cluster-autoscaler/integration/framework.go

```diff
@@ -206,3 +243,115 @@ func (driver *Driver) scaleWorkload(workloadName string, replicas int32) error {
 	}
 	return nil
 }
+
+func (driver *Driver) getFirstNode() (*v1.Node, error) {
+	nodeList, err := driver.targetCluster.Clientset.CoreV1().Nodes().List(context.Background(), metav1.ListOptions{})
+	if err != nil {
+		return &v1.Node{}, err
+	}
+	sort.Slice(nodeList.Items, func(i, j int) bool {
+		return nodeList.Items[i].ObjectMeta.CreationTimestamp.Before(&nodeList.Items[j].ObjectMeta.CreationTimestamp)
+	})
+	latestNode := nodeList.Items[0]
```

Comment:
Nit: You might want to name this object as `firstNode`. Otherwise, it will be confusing while reading the code.

Reply:
Thanks, the changes are done in the [PR review Commit.](https://github.com/gardener/autoscaler/pull/117/commits/c41b337bb635f451312821595e0d46a28e69f7c6)